.github/workflows/publish-containers.yml

```diff
@@ -0,0 +1,55 @@
+name: Publish containers
+
+# "on":
+#   push:
+#     tags:
+#       - "v[0-9]+.[0-9]+.[0-9]+"
+#       - "v[0-9]+.[0-9]+.[0-9]+-pre[0-9]+"
+
+on:
+  pull_request:
+jobs:
+  master:
+    strategy:
+      matrix:
+        version: ['0.0.1']
```

Comment:
it's better if you track version in git in form of a tag and read it with `git tag` or `git descirbe`